Modules/HTTP/Tests/HTTPTests/Message/RequestTests.swift

```diff
@@ -10,16 +10,16 @@ public class RequestTests : XCTestCase {
         XCTAssertEqual(request.headers, ["Content-Length": "0"])
         XCTAssertEqual(request.body, .buffer(Buffer()))
 
-        request = Request(body: Drain(buffer: "foo") as Core.InputStream)
+        request = Request(body: BufferStream(buffer: "foo"))
         XCTAssertEqual(request.method, .get)
         XCTAssertEqual(request.url, URL(string: "/"))
         XCTAssertEqual(request.version, Version(major: 1, minor: 1))
         XCTAssertEqual(request.headers, ["Transfer-Encoding": "chunked"])
         XCTAssertTrue(request.body.isReader)
 
         request = Request { stream in
-            try stream.write("foo")
-            try stream.flush()
+            try stream.write("foo", deadline: 1.second.fromNow())
```

Comment:
Why the `fromNow` all over the place? Can't it be injected at the landing site? I'm sure that a lot of people will misuse these APIs (especially since its not documented anywhere).